docs/components/stepper/index.html

```diff
@@ -49,7 +47,7 @@ <h2 id="basic-stepper">Basic Stepper</h2>
               <span class="beta-hxStepLabel">Topic</span>
               <span class="beta-hxStepValue">Technical Incident</span>
             </header>
-            <div class="hxBox hxMd">
+            <div class="hxBox hxMd beta-hxContainerStyle">
```

Comment:
We shouldn't need the extra class.  Please remove.

Reply:
Changes have been done, please review.